base/mem_trie.hpp

```diff
@@ -382,7 +382,7 @@ class MemTrie
     {
       ASSERT_LESS_OR_EQUAL(n, Size(), ());
 
-      Edge const prefix(m_label.rbegin(), m_label.rbegin() + n);
+      Edge prefix(m_label.rbegin(), m_label.rbegin() + n);
```

Comment:
FYI: As practice shows, modern compilers are smart enough to still apply RVO in these cases.